docs/Using-Virtual-Environment.md

```diff
@@ -1,39 +1,52 @@
 # Using Virtual Environment
 
 ## What is a Virtual Environment?
-A Virtual Environment is a self contained directory tree that contains a Python installation
-for a particular version of Python, plus a number of additional packages. To learn more about
-Virtual Environments see [here](https://docs.python.org/3/library/venv.html).
+
+A Virtual Environment is a self contained directory tree that contains a Python
```

Comment:
@mmattar Are you setting a new default line width for markdown files? If yes, how is it ensured that all files follow the same convention?

Reply:
This is a one-pass clean-up. Afterward, we'll evaluate if we use Prettier or not. We have a pre-commit config that we'll enable.

Reply:
If we decide not to use prettier in the future, how do you plan to undo changes which have been incorporated by prettier in this PR?

Reply:
We will either use prettier, another tool or continue doing a best-effort. What we get out of using Prettier now is defining the standard and rolling it out.

Reply:
sounds good. Are we apply prettier formatting to all markdown files instead of just a handful for now?

Reply:
One PR at a time. I'll essentially be touching most of the files due to the restructuring.